core/rawdb/accessors_chain.go

```diff
@@ -1544,32 +1543,75 @@ func DeleteInboundEtxs(db ethdb.KeyValueWriter, hash common.Hash) {
 	}
 }
 
-func WriteAddressUtxos(db ethdb.KeyValueWriter, address common.Address, utxos []*types.UtxoEntry) {
-	data, err := rlp.EncodeToBytes(utxos)
-	if err != nil {
-		log.Global.WithField("err", err).Fatal("Failed to rlp encode utxos")
+func WriteAddressOutpoints(db ethdb.KeyValueWriter, outpointMap map[string][]*types.OutpointAndDenomination) error {
```

Comment:
See https://github.com/dominant-strategies/go-quai/pull/1579/commits/9fec33e9c9652502e4ec03ec5fa8b55a09f70574#r1565896992

The outpointMap should be a map of maps, not a map of arrays